hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/main/java/org/apache/hadoop/yarn/server/router/clientrm/RouterYarnClientUtils.java

```diff
@@ -17,17 +17,14 @@
  */
 package org.apache.hadoop.yarn.server.router.clientrm;
 
-import java.util.Collection;
-import java.util.HashMap;
-import java.util.Map;
+import java.util.*;
```

Comment:
Avoid messing too much with the imports.

Reply:
Thanks for the reminder, I'll fix it.